frontend/src/app/PrimeErrorBoundary.tsx

```diff
@@ -37,6 +46,15 @@ export default class PrimeErrorBoundary extends React.Component<
       return <ErrorPage />;
     }
 
+    if (this.state.redirectToOkta) {
+      window.location.href = `${
+        process.env.REACT_APP_OKTA_URL
+      }/oauth2/default/v1/authorize?client_id=${
+        process.env.REACT_APP_OKTA_CLIENT_ID
+      }&redirect_uri=${getUrl()}&response_type=token id_token&scope=openid simple_report simple_report_test&nonce=thisisnotsafe&state=thisisbogus`;
```

Comment:
It would be a good idea to uri encode the query parameter values here even though with these particular values it looks like it would work.

I don't think we want a static `state` and `nonce` here, do we?  Is there a reason the test scope is always included instead of the environment-specific ones?

Reply:
Yeah, the scope thing was my bad :cry: fix is here:
https://github.com/CDCgov/prime-simplereport/pull/2779

I can do URI encoding in that PR as well